src/Tests/Tests.MonoTorrent.IntegrationTests/IntegrationTests_RealPieceWriter.cs

```diff
@@ -232,7 +238,7 @@ public async Task CreateAndDownloadTorrent (TorrentType torrentType, bool create
             var fileSource = new TorrentFileSource (_seederDir.FullName);
             fileSource.TorrentName = _torrentName;
             TorrentCreator torrentCreator = new TorrentCreator (torrentType);
-            torrentCreator.Announce = $"http://{new IPEndPoint (LoopbackAddress, _trackerPort)}/announce";
+            //torrentCreator.Announce = $"http://{new IPEndPoint (LoopbackAddress, _trackerPort)}/announce";
```

Comment:
restore this

Reply:
add retry/backoff when a connection cannot be established.

test fails because all peer connections ran while seeder was hashing